### ISSUE TITLE
SeleniumInterface - Add CheckExist

### ISSUE THREAD

**Issue by AsafSilman** [AsafSilman/testbuilder#4]
Add CheckExist functionality to interface

**Comment by AsafSilman** [comment]
```
    @action_word
    def CheckExist(self, step_context):
        xpath = step_context.step_argument_1_mapped

        temp_wait = self.implicit_wait # Maintain this value
        
        self.set_implicit_wait(temp_wait/2)
        condition = all([
            self.wait_for_element_condition("element_to_be_clickable", xpath),
            self.wait_for_element_condition("visibility_of_element_located", xpath),
            self.wait_for_element_condition("presence_of_element_located", xpath)
        ])
        self.set_implicit_wait(temp_wait)

        step_context.step.result = condition
```

**Comment by AsafSilman** [comment]
Closed by #8
